src/components/UserInformation.js

```diff
@@ -11,6 +11,11 @@ const usernameOptions = [
   { value: 'eracommons', label: 'eRA Commons ID' }
 ];
 
+// required for any user creation:
+const requiredFields = ['name', 'usernameOption'];
+// required when a DAC creates a PI:
+const requiredDacFields = ['organization', 'expiration'];
```

Comment:
it seems like these two are only used together, is there a reason to have them separated?

Reply:
ah, right, this doesn't need to be separate anymore